src/playlist/playlistcontainer.cpp

```diff
@@ -363,6 +364,10 @@ void PlaylistContainer::SavePlaylist(const int id) {
 
 }
 
+void PlaylistContainer::SaveAllPlaylists() {
```

Comment:
This slot function is unnecessary as you can connect directly to `PlaylistManager::SaveAllPlaylists`